### ISSUE TITLE
Support fine-tuned OpenAI models

### ISSUE THREAD

**Issue by Wehzie** [promptfoo/promptfoo#267]
Fine-tuned models may carry identifies such as `ft:gpt-3.5-turbo-0613:company-name::ID`. Currently, promptfoo doesn't seem to parse these identifiers correctly.

I've attempted the following variations.
```
  - openai:ft:gpt-3.5-turbo-0613:company-name::ID:
      id: gpt-3.5-custom
  - openai:chat:ft:gpt-3.5-turbo-0613:company-name::ID:
      id: gpt-3.5-custom
  - openai:completion:ft:gpt-3.5-turbo-0613:company-name::ID:
      id: gpt-3.5-custom
```

An example of an error message.
```
[FAIL] API response error: TypeError: Cannot read properties of undefined (reading '0'): {"error":{"message":"The model `ft` does not exist","type":"invalid_request_error","param":null,"code":"model_not_found"}}
```

It seems that name parsing stops after "ft".

**Comment by typpo** [comment]
Thanks for the report.  b52de61c6e1fd0a9e67d2476a9f3f9153084ad61 should fix this.  

The correct format would be `openai:chat:ft:gpt-3.5-turbo-0613:company-name::ID:` or `openai:completion:ft:gpt-3.5-turbo-0613:company-name::ID:`, depending on whether the model uses the `chat`-format or `completion`-format prompt.

**Comment by typpo** [comment]
Fixed in 0.26.2 release
